integrationtest/integration_test.go

```diff
@@ -220,6 +243,84 @@ func TestFailWithBadTokenReviewerJwt(t *testing.T) {
 	}
 }
 
+func TestAuthAliasMetadataAssignment(t *testing.T) {
+	// annotate the service account
+	expMetadata := map[string]string{
+		"foo": "bar",
+		"bar": "baz",
+	}
+
+	annotationPrefix := "auth-metadata.vault.hashicorp.com/"
```

Comment:
nit: Makes the intent a little clearer

```suggestion
	const annotationPrefix = "auth-metadata.vault.hashicorp.com/"
```